source/guides/clear/mixer.rst

```diff
@@ -284,79 +312,107 @@ set to get a smaller kernel image, which will also be faster to load.
 
-#. Configure image. Edit the ister configuration file for your image to include
-   all of the bundles you want preinstalled in the image. If this is the first
-   time creating an image, first get a copy of the
-   :file:`release-image-config.json` template file:
+Example 3: Create an update for your mix
+========================================
+
+Next, let’s create a new version of the mix. We’ll add a new bundle.
+
+#. Create a new version of your mix, for the live image to
+   update to. Increment your mix version by 10:
 
    .. code-block:: bash
 
-      curl -O https://raw.githubusercontent.com/bryteise/ister/master/release-image-config.json
+      mixer versions update
 
-   For this example, edit :file:`release-image-config.json` so that the root
-   partition size is "5G" and replace the "kernel-native" bundle with
-   "kernel-kvm".
+#. Add the upstream :command:`curl` bundle to version 20 of the mix:
 
-   .. code-block:: console
+   .. code-block:: bash
 
-      {
-        "DestinationType" : "virtual",
-        "PartitionLayout" : [ { "disk" : "release.img", "partition" : 1, "size" : "32M", "type" : "EFI" },
-                              { "disk" : "release.img", "partition" : 2, "size" : "16M", "type" : "swap" },
-                              { "disk" : "release.img", "partition" : 3, "size" : "5G", "type" : "linux" } ],
-        "FilesystemTypes" : [ { "disk" : "release.img", "partition" : 1, "type" : "vfat" },
-                              { "disk" : "release.img", "partition" : 2, "type" : "swap" },
-                              { "disk" : "release.img", "partition" : 3, "type" : "ext4" } ],
-        "PartitionMountPoints" : [ { "disk" : "release.img", "partition" : 1, "mount" : "/boot" },
-                                   { "disk" : "release.img", "partition" : 3, "mount" : "/" } ],
-        "Version": "latest",
-        "Bundles": ["kernel-kvm", "os-core", "os-core-update"]
-      }
+      mixer bundle add curl
 
-#. Build the image.
+#. Build your next mix version that incorporates the new bundle.
 
    .. code-block:: bash
 
-      sudo mixer build image
-
-   The output from this step will be :file:`release.img`, which is a live image.
+      mixer build bundles
+      mixer build update
 
-#. Make the next mix. Create a new version of your mix, for the live image to
-   update to. Increment your mix version by 10:
+#. Optionally, you can build delta-packs, which help reduce client update
+   time:
 
    .. code-block:: bash
 
-      mixer versions update
+      mixer build delta-packs --from 10 --to 20
+
+Refresh your \http://localhost site to see the update content for
+mix version 20.
+
+You can also look in ~/mixer/update/www/<mix version> to see the update
+content in your workspace.
+
+
+Example 4: Build an image
+=========================
+
+This example shows how to build an image incorporating the "kernel-kvm”
+bundle from Example 2, using `ister`_.
```

Comment:
This is what I should suggest fro lines 359-360:

"This example shows how to build a bootable image containing the "kernel-kvm”, "os-core", and the "os-core-update" bundles from Example 2.  Underneath, mixer uses `ister`_ to generate the image. "

Reply:
Good revision.